File: app/src/main/java/com/tstudioz/fax/fme/database/LeanTask.java
package com.tstudioz.fax.fme.database;

import java.util.Date;

import io.realm.RealmObject;

public class LeanTask extends RealmObject {

    public String taskTekst;
    public Date dateCreated;
    public Date dateReminder;
    public Boolean reminder;

    public void setDateCreated(Date dateCreated) {
        this.dateCreated = dateCreated;
    }

    public void setDateReminder(Date dateReminder) {
        this.dateReminder = dateReminder;
    }

    public void setReminder(Boolean reminder) {
        this.reminder = reminder;
    }

    public void setTaskTekst(String taskTekst) {
        this.taskTekst = taskTekst;
    }

    public Boolean getReminder() {
        return reminder;
    }

    public Date getDateCreated() {
        return dateCreated;
    }

    public Date getDateReminder() {
        return dateReminder;
    }

    public String getTaskTekst() {
        return taskTekst;
    }
}

File: app/src/main/java/com/tstudioz/fax/fme/database/LeanTask.kt
package com.tstudioz.fax.fme.database

import io.realm.kotlin.types.RealmObject
import io.realm.kotlin.types.annotations.Ignore
import io.realm.kotlin.types.annotations.PrimaryKey
import java.util.Date

open class LeanTask : RealmObject {

    @PrimaryKey
    var id: String? = null
    var taskTekst: String? = null
    @Ignore
    var dateCreated: Date? = null
    @Ignore
    var dateReminder: Date? = null
    var reminder: Boolean? = null
    var checked: Boolean = false

}
